Cargo.toml

```diff
@@ -24,7 +24,7 @@ bitcoind = {version = "0.32", features = ["25_0"] }
 
 #Empty default feature set, (helpful to generalise in github actions)
 [features]
-default = []
+default = ["integration-test"]
```

Comment:
It seems to me we don't need the `integration-test` feature at all. As usual I forgot why I added it. probably some dependency requirements that got solved afterward.